### ISSUE TITLE
dotnet tool support

### ISSUE THREAD

**Issue by ErikApption** [microsoft/NugetNinja#61]
Any way to support this project to be deployed into nuget as dotnet tool? https://learn.microsoft.com/en-us/dotnet/core/tools/global-tools-how-to-create. Don't mind sending a PR for the github workflows, but is there anything else that could be done?

**Comment by Anduin2017** [comment]
Yes and actually I meant to do that. However due to Microsoft compliance rules currently I couldn't figure out how I can publish it on behalf of Microsoft identity yet. I will dig deeper for achieving that.
